src/main/java/app/studnicki/ox/Score.java

```diff
@@ -0,0 +1,34 @@
+package app.studnicki.ox;
+
+class Score {
+  int value;
+
+  private static final int LIMIT = 7;
+  private static final int WIN = 3;
+  private static final int DRAW = 1;
```

Comment:
I've met this before. Why constants (static)?

Reply:
I think that is the good way. All of the Score instances will be using the same values. That is why static final.